### PR TITLE
Meeting notes for Operator SIG meeting June 2023

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: operators/meetings/040-20230608.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+the airplane industry. They highlighted the need for a standardized way to represent
+models and identified ONNX as a suitable candidate. However, they noted that ONNX's 
+existing operator specifications are not strict enough for their applications. 
+They require a detailed definition of all operators with precise mathematical formulas.
+
+The potential collaboration with Airbus is projected to last a few years and the company
+has expressed their willingness to contribute. However, there may be additional legal
+restrictions concerning standards utilized in the aviation industry. 
+
+I would be beneficial to the ONNX standard to define operators strictly, especially if
```
Comment (gramalingam) [inline operators/meetings/040-20230608.md:42]: "I" to "It"

File: operators/meetings/040-20230608.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+### `SlidingWindowInferer` pre- and post-processing operator
+
+In response to the issue (see [Issue #5270](https://github.com/onnx/onnx/issues/5270)), 
+Liqun Fu has expressed willingness to create a PR for the `SlidingWindowInferer` 
+pre- and post-processing (PNP) operator. 
+
+This operator plays an important role in computer vision applications, where models
+are usually trained with input data divided into patches due to GPU memory limitations.
+During the inference stage, input images are also divided into patches.
+Patrial outputs from a model are then aggregated with an importance weight
```
Comment (gramalingam) [inline operators/meetings/040-20230608.md:78]: "Partial"

File: operators/meetings/040-20230608.md (hunk 1 above)
Comment (postrational) [reply operators/meetings/040-20230608.md]: Fixed.

File: operators/meetings/040-20230608.md (hunk 2 above)
Comment (postrational) [reply operators/meetings/040-20230608.md]: Fixed.

**Review by gramalingam — APPROVED** [review]

**Pull request merged**